apps/web/pages/index.tsx

```diff
@@ -174,7 +151,18 @@ const Home: NextPage = () => {
       </Heading>
 
       <Flex flexDirection="column" gap={4}>
-        {subscriptionData?.isSubscribed ? (
+        {isLoading || w3iClientIsLoading || !w3iClient ? (
```

Comment:
I'll be updating the loading states to have better handling and to give better feedback to the user on the example app. The users of the hooks should be aware of that there might be different loading states and they should render different outputs depending on the loading type (account, client, subscription, etc)